projectile.el

```diff
@@ -4447,11 +4447,15 @@ project of that type"
       (projectile-read-command prompt default-cmd)
     default-cmd))
 
+(defcustom projectile-comint-mode nil
+  "If non-nil, launch projectile compilation buffers in interactive mode."
+  :type 'bool)
+
 (defun projectile-run-compilation (cmd)
```

Comment:
I think you can also add here an optional argument called `comint-mode`, so that people can triggered potentially with a prefix arg from interactive commands.

Reply:
I would, but the user level function is `projectile-compile-project`, which already has a prefix argument (used to force the command prompt). Should I remove it in favour of this one ?